src/goto-symex/path_storage.cpp

```diff
@@ -19,6 +19,14 @@ nondet_symbol_exprt symex_nondet_generatort::operator()(const typet &type)
   return nondet_symbol_exprt(identifier, type);
 }
 
+nondet_symbol_exprt symex_nondet_generatort::operator()(
+  typet type, source_locationt location)
+{
+  irep_idt identifier = "symex::nondet" + std::to_string(nondet_count++);
```

Comment:
Factor this with the function above into `fresh_symex_nondet_identifier` or something, so we don't accidentally change one but not the other in future

Reply:
See below: I think the above should just be removed.